infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -36,6 +36,7 @@ public class StatusLogger {
 
   public static final StatusLogger STATUS_LOG =
       new StatusLogger(LoggingConfigurator.STATUS_LOGGER_NAME);
+  public static final int KEY_LIMIT = 20;
```

Comment:
Just a nit but this can be private, also maybe can rename to be more descriptive to `PUBKEY_LOG_LIMIT`